lib/screens/login/login_repository.dart

```diff
@@ -12,14 +15,50 @@ class FakeLoginRepository implements LoginRepository {
   @override
   Future<String> login(String email, String password) async {
     // Simulate network delay
+    SharedPreferences sharedPreferences = await SharedPreferences.getInstance();
+
     await Future.delayed(Duration(seconds: 1));
 
     if (Random().nextBool()) {
       // random network error
       throw NetworkException();
     } else {
       // fake successful login
-      return "Token abcdEfgh";
+      sharedPreferences.setString(S.auth, "Token 9935a8b04f2de7f5dec8f9e92a1893822b034dc7");
```

Comment:
Not to be done here. Since this is a common step b/w Fake and API repo. Also, more importantly, this step doesnot belong to usecase of **Repository**. It is a screen logic, should exist in cubit.
remove it.